core/amber/src/test/scala/edu/uci/ics/amber/engine/e2e/PauseSpec.scala

```diff
@@ -58,11 +58,13 @@ class PauseSpec
     parent.expectMsg(ControllerState.Running)
     controller ! ControlInvocation(AsyncRPCClient.IgnoreReply, PauseWorkflow())
     parent.expectMsg(ControllerState.Paused)
+    Thread.sleep(4000)
```

Comment:
what is this sleeping for?

Reply:
This is for ensuring the workflow is not progressing after a pause. If the workflow is still running, after 4s it should be in a completed state.